devices/ak2i.cpp

```diff
@@ -1,5 +1,5 @@
-#include "device.h"
-#include "delay.h"
+#include "../device.h"
```

Comment:
please don't use relative paths

Reply:
Sorry about that. I do that to appease my editor's linter, which doesn't know what directories are include directories and the like.

(And quite honestly, I'd prefer relative paths for some things, this being one of them, but I'll do my best to keep them clear.)